README.md

```diff
@@ -8,28 +8,33 @@
 
 require in the SDK:
 
-`const OneblinkSDK = require('@oneblink/sdk')`
+````javascript
+const OneblinkSDK = require('@oneblink/sdk')
+```
 
 initialise the SDK object:
 
 ```javascript
-const oneblink = new OneblinkSDK()
+const FORMS_KEY = '12345'
+const FORMS_SECRET = 'abcd'
+const oneblink = new OneblinkSDK({
```

Comment:
Do we maybe want to go more like AWS and have `new OneBlink.Forms()` as the constructor?